envs/development/conda/fedbiomed-gui.yaml

```diff
@@ -51,5 +51,5 @@ dependencies:
       # FLamby
       - git+https://github.com/owkin/FLamby@main
       # declearn
-      - declearn[torch1] ~=2.3.2
+      - declearn[torch1] ~=2.5.0
```

Comment:
now that declearn v.2.6 has been released, could we specify it? or does it break compatibility with Fed-BioMed?